modules/features2d/include/opencv2/features2d.hpp

```diff
@@ -719,13 +723,16 @@ class CV_EXPORTS_W SimpleBlobDetector : public Feature2D
       CV_PROP_RW bool filterByConvexity;
       CV_PROP_RW float minConvexity, maxConvexity;
 
+      CV_PROP_RW bool collectContours;
+
       void read( const FileNode& fn );
       void write( FileStorage& fs ) const;
   };
 
   CV_WRAP static Ptr<SimpleBlobDetector>
     create(const SimpleBlobDetector::Params &parameters = SimpleBlobDetector::Params());
   CV_WRAP virtual String getDefaultName() const CV_OVERRIDE;
+  CV_WRAP virtual const std::vector<std::vector<Point> >& getBlobContours() const = 0;
```

Comment:
"Source API" compatibility doesn't allow to add new virtual **pure** methods.

Do we really want to violate that and bypass checker?

---

- 4.x: implementation with `CV_Error(not implemented)` should be used instead.
- 5.x: pure method patch should be used

Reply:
@alalek I added stup implementation with `CV_Error`.